### ISSUE TITLE
DHCP lease appear multiple time in db and block the service

### ISSUE THREAD

**Issue by Kiroha** [AdguardTeam/AdGuardHome#5219]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Other (please mention in the description)

### CPU architecture

x86

### Installation

Docker

### Setup

Other (please mention in the description)

### AdGuard Home version

0.107.19

#### What did you do?
Activate DHCP server on Adguard
#### Expected result
DHCP server correctly affect lease to machine
#### Actual result
DHCP server affect lease multiple time inside the DB 
#### Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/28816263/204913400-1716ff20-cc6c-4d10-ae6e-60f0d7e52b40.png)

#### Additional information
I've completely reset my container yesterday during outage cause don't know the root cause. So I've deploy a fresh new Adguard home and activate DHCP and since I didn't suceed to have a DHCP working properly.

**Comment by marcpearson** [comment]
Same problem since update to version 0.107.22. Was working fine on 0.107.21. I'm running AdguardHome in a jail on FreeBSD

**Comment by Kiroha** [comment]
Yep solve for me also. I think the version with the DHCP Lease inside a DB file (if I'm not mistaken) being kept through update help a lot
